geoutils/projtools.py

```diff
@@ -48,6 +48,41 @@ def bounds2poly(boundsGeom, in_crs=None, out_crs=None):
     return bbox
 
 
+def merge_bounds(bounds_list, merging_algorithm="union"):
+    """
+    Merge a list of bounds into single bounds, using either the union or intersection.
+
+    :param bounds_list: A list of geometries with bounds, i.e. a list of coordinates (xmin, ymin, xmax, ymax), \
+a rasterio/Raster object, a geoPandas/Vector object.
+    :type bounds_list: list, tuple
```

Comment:
Do you mean a list of tuples? This is the same as `Union[list, tuple]`. Unfortunately, I don't know how it should rather be..

EDIT: Or am I wrong? Do you know where the documentation is for docstring type annotations (btw, we should switch sooner than later..)